db.go

```diff
@@ -63,6 +63,9 @@ type DB struct {
 	// https://github.com/boltdb/bolt/issues/284
 	NoGrowSync bool
 
+	// When true, Linux 2.6.23+ sets MAP_POPULATE for sequential read-ahead.
+	MmapFlags bool
```

Comment:
Then in the comment we can mention that if you want to read entire database fast, you can set MmapFlag to `syscall.MAP_POPULATE` on linux.